dts/bindings/gpio/ti,boosterpack-header.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 Brett Witherspoon
+# SPDX-License-Identifier: Apache-2.0
+
+title: TI BoosterPack GPIO header
+
+description: |
+    GPIO pins exposed as BoosterPack headers on TI LaunchPads.
+
+    BoosterPack plug-in modules are available in 20 and 40 pin variants. The
+    20 pin variant has two 10 x 1 pin headers and the 40 pin variant has two
+    10 x 2 pin headers. Both variants are compatible and stackable.
+
+    The pins of the 20 pin variant and the outer row of the 40 pin variant are
+    numbered 1 through 20. The inner rows of the 40 pin variant are numbered 21
+    through 40. The BoosterPack pinout is depicted below:
+
+    1  3.3V     21 5V                    40 GPIO 20 GND
+    2  Analog   22 GND                   39 GPIO 19 GPIO / SPI CS
```

Comment:
Pin 18 should be "GPIO/SPI CS", not pin 19.

Reply:
One off. Fixed.